port_status_yaml.py

```diff
@@ -18,10 +18,17 @@ class Comment:
     class Source:
         repo: str
         commit: str
+    @dataclass
+    class Label:
+        name: str
+        color: str
+        text_color: str = '000000'
     ported: bool
     source: Optional[Source]
     mathlib4_pr: Optional[int]
     mathlib4_file: Optional[str]
+    labels: Optional[list[Label]]
+    sync_prs: Optional[list[int]]
```

Comment:
These aren't optional (in the python meaning of "can be `None`"), but if absent the semantics are an empty list
```suggestion
    labels: list[Label] = field(default_factory=list)
    sync_prs: list[int] = field(default_factory=list)
```